code/test/util.js

```diff
@@ -245,8 +415,8 @@ const commitReveal = async ({ layers, Debugger, index, eotp, paramsHash, commitP
   Logger.debug(`Revealing`, { authParams, revealParams })
   const wouldSucceed = await wallet.reveal.call(authParams, revealParams)
   Logger.debug(`Reveal success prediction`, !!wouldSucceed)
-  const tx = await wallet.reveal(authParams, revealParams)
-  return { tx, authParams, revealParams }
```

Comment:
why is tx not returned? Many tests in spendLimit and innerCore rely on tx

Reply:
Apologies, I removed when debugging something and forgot to add it back in.
Fixed now 🙏